src/compute-pr-actions.ts

```diff
@@ -318,9 +318,9 @@ export function process(prInfo: BotNotFail,
     // Needs author attention (bad CI, merge conflicts)
     if (info.needsAuthorAction) {
         context.targetColumn = "Needs Author Action";
-        if (info.hasMergeConflict) post(Comments.MergeConflicted(info.headCommitAbbrOid, info.author));
-        if (info.failedCI) post(Comments.CIFailed(info.headCommitAbbrOid, info.author, info.ciUrl!));
-        if (info.hasChangereqs) post(Comments.ChangesRequest(info.headCommitAbbrOid, info.author));
+        if (info.hasMergeConflict) post(Comments.MergeConflicted(info.headCommitOid, info.author));
+        if (info.failedCI) post(Comments.CIFailed(info.headCommitOid, info.author, info.ciUrl!));
+        if (info.hasChangereqs) post(Comments.ChangesRequest(info.headCommitOid, info.author));

```

Comment:
`info` here is the extended one, so if you add a `headCommitAbbrOid` field to it (in `extendPrInfo`), then this code would stay the same (and most of the below).

Reply:
Done :heavy_check_mark: